wagtail_transfer/field_adapters.py

```diff
@@ -213,8 +215,13 @@ def __init__(self, field):
         self.related_base_model = get_base_model(field.related_model)
         self.is_parental = isinstance(self.related_field, ParentalKey)
         self.is_followed = (get_base_model(self.field.model)._meta.label_lower, self.name) in FOLLOWED_REVERSE_RELATIONS
+        logger.debug(f"Checking field adaptor: {self.name}, {get_base_model(self.field.model)}', {get_base_model(field.related_model)}")
 
     def _get_related_objects(self, instance):
+        if getattr(instance, self.name).all():
```

Comment:
This if/else feels ugly, but I ran into problems with unit test failures where `test_import_page_with_comments` failed because the comment instance didn't appear to have a page associated with it (or something similar). 

I wasn't 100% sure it wasn't an error with the test fixture data but I think it's more likely that I just didn't fully understand how it was supposed to be used.